modules/renter/contractor/dependencies.go

```diff
@@ -68,32 +67,43 @@ type walletBridge struct {
 func (ws *walletBridge) NextAddress() (types.UnlockConditions, error) { return ws.w.NextAddress() }
 func (ws *walletBridge) StartTransaction() transactionBuilder         { return ws.w.StartTransaction() }
 
-// stdPersist implements the persister interface via persist.SaveFile and
-// persist.LoadFile. The metadata and filename required by these functions is
-// internal to stdPersist.
+// stdPersist implements the persister interface via jj.OpenJournal and
+// jj.CheckPoint. The filename required by these functions is internal to
+// stdPersist.
 type stdPersist struct {
-	meta     persist.Metadata
+	journal  *journal
 	filename string
 }
 
-func (p *stdPersist) save(data contractorPersist) error {
-	return persist.SaveFile(p.meta, data, p.filename)
+func (p stdPersist) save(data contractorPersist) error {
+	if p.journal == nil {
+		var err error
+		p.journal, err = openJournal(p.filename, data)
+		return err
+	}
+	return p.journal.checkpoint(data)
```

Comment:
This is a sync operation? I would like to move the project in the direction of always syncing.